core/src/main/java/org/frankframework/validation/SchemaUtils.java

```diff
@@ -86,6 +87,26 @@ public class SchemaUtils {
 
 	public static final QName WSDL_SCHEMA = new QName(XSD, "schema", "");
 
+	/**
+	 * Check if the StartElement is one of given qnames.
+	 */
+	public static boolean isElement(StartElement el, QName... names) {
```

Comment:
Ik vind deze overload niet echt wat toevoegen/verduidelijken. Kan je niet beter direct de `isName` gebruiken?

Reply:
De overloads met `StartElement` / `EndElement` zijn voor mij juist de duidelijkste in gebruik, en degene die het meest nodig zijn. Maar ja, het was bedoelt als kleine verbetering aan code en als we het niet eens kunnen worden over wat beter is dan slaagt het PR daar niet in 😅